tox.ini

```diff
@@ -34,6 +34,34 @@ deps =
     # https://github.com/pytest-dev/pytest-rerunfailures/issues/134
     # pytest @ git+https://github.com/pytest-dev/pytest.git
 
+[testenv:packaging]
+description =
+    Do packaging/distribution. If tag is not present or PEP440 compliant upload to
+    PYPI could fail
+# `usedevelop = true` overrides `skip_install` instruction, it's unwanted
+usedevelop = false
+# don't install package in this env
+skip_install = true
+deps =
+    collective.checkdocs >= 0.2
+    pep517 >= 0.8.2
+    pip >= 20.2.2
+    toml >= 0.10.1
```

Comment:
This is a transitive dependency of `pep517`, it probably doesn't matter what version it is.